Makefile

```diff
@@ -82,9 +82,12 @@ build-no-cache: dist
 shell: build ## start a shell inside the build env
 	$(DOCKER_RUN_TRAEFIK) /bin/bash
 
-image: binary ## build a docker traefik image
+image: clear-static binary ## build a docker traefik image
 	docker build -t $(TRAEFIK_IMAGE) .
 
+clear-static:
+	rm -rf static
```

Comment:
Just to be sure: When the static folder does not exist, will the make file continue nicely (and not produce any disturbing output)?